sql/core/src/main/scala/org/apache/spark/sql/execution/window/WindowExec.scala

```diff
@@ -46,7 +46,7 @@ import org.apache.spark.sql.types.{CalendarIntervalType, DateType, IntegerType,
  *   rows to this frame.
  * - Moving frame: Every time we move to a new row to process, we remove some rows from the frame
  *   and we add some rows to the frame. Examples are:
- *     1 PRECEDING AND CURRENT ROW and 1 FOLLOWING AND 2 FOLLOWING.
```

Comment:
This is a valid window frame.

Reply:
Are you sure?
I seems mean:
`PRECEDING AND CURRENT ROW`
and
`1 FOLLOWING AND 2 FOLLOWING`

Reply:
Feel free to try the syntax :). I am pretty sure that both are valid.

Reply:
Thanks. It's OK.